pkg/config/analysis/msg/messages.gen.go

```diff
@@ -29,9 +29,9 @@ var (
 	// Description: A pod is missing the Istio proxy.
 	PodMissingProxy = diag.NewMessageType(diag.Warning, "IST0103", "The pod %s is missing the Istio proxy. This can often be resolved by restarting or redeploying the workload.")
 
-	// GatewayPortNotOnWorkload defines a diag.MessageType for message "GatewayPortNotOnWorkload".
-	// Description: Unhandled gateway port
-	GatewayPortNotOnWorkload = diag.NewMessageType(diag.Warning, "IST0104", "The gateway refers to a port that is not exposed on the workload (pod selector %s; port %d)")
+	// GatewayPortNotDefinedOnService defines a diag.MessageType for message "GatewayPortNotDefinedOnService".
```

Comment:
@ericvn @dhawton will this require a separate PR to update https://istio.io/latest/docs/reference/config/analysis/ist0104/, or are these pages generated automatically?

Reply:
This would be a separate PR. That page is not generated

Reply:
I think only https://github.com/istio/istio.io/blob/master/data/analysis.yaml will be synced by istio-testing once this PR is merged, the config page needs to be revised as well.

Reply:
This is a great reminder. I forgot about synchronizing the documentation.

Reply:
New related documents PR： https://github.com/istio/istio.io/pull/13800

Reply:
@GregHanson PTAL, thanks